lisa/tools/xfstests.py

```diff
@@ -128,9 +128,13 @@ def _install_dep(self) -> None:
 
         # if install the packages in one command, the remain available packages can't
         # be installed if one of packages is not available in that distro,
-        # so here install it one by one.
+        # so here install it one by one
         for package in list(package_list):
-            posix_os.install_packages(package)
+            # to make code simple, put all packages needed by one distro in one list.
+            # the package name may be different for the different sku of the
+            #  same distro. so, install it when the package exists in the repo.
+            if posix_os.is_package_in_repo(package):
+                posix_os.install_packages(package)
```

Comment:
Another way is to add a boolean on `install_packages` called `ignore_non_exists`. It makes the logic simpler, and easy to read.

Reply:
I will raise another PR for it, use #1641 to track.